requirements.txt

```diff
@@ -0,0 +1,19 @@
+aniso8601==9.0.1 ; python_version >= "3.9" and python_version < "4.0"
```

Comment:
I've not seen these `python_version` restrictions before - if I'm reading rightly this says:
> install `aniso8601==9.0.1` _if_ python version >= 3.9 and < 4.0

Does that mean that this "succeeds" but doesn't actually install anything if you've got python 3.8? This seems weird.. can we just drop those, or am I missing a strong reason to want them?

Edit: ran it to check, yes it just goes:
> Ignoring zipp: markers 'python_version >= "3.9" and python_version < "3.10"' don't match your environment

Reply:
I'm actually not too sure honestly. I'm happy to keep or remove as per whatever we think is best ^^;